diff --git a/compat/cuda/ptx2c.sh b/compat/cuda/ptx2c.sh
index 48452379c2..1c486dc30e 100755
--- a/compat/cuda/ptx2c.sh
+++ b/compat/cuda/ptx2c.sh
@@ -26,9 +26,10 @@ OUT="$1"
 IN="$2"
 NAME="$(basename "$IN" | sed 's/\..*//')"
 
-printf "const char %s_ptx[] = \\" "$NAME" > "$OUT"
+printf "const char %s_ptx[] = {\\" "$NAME" > "$OUT"
 echo >> "$OUT"
-sed -e "$(printf 's/\r//g')" -e 's/["\\]/\\&/g' -e "$(printf 's/^/\t"/')" -e 's/$/\\n"/' < "$IN" >> "$OUT"
-echo ";" >> "$OUT"
+xxd -i < "$IN" >> "$OUT"
+echo "  ,0x00" >> "$OUT"
+echo "};" >> "$OUT"
 
 exit 0
diff --git a/configure b/configure
index fb55e04ee7..f358edfb3f 100755
--- a/configure
+++ b/configure
@@ -1050,6 +1050,7 @@ test_nvcc(){
     tmpo_=$TMPO
     [ -x "$(command -v cygpath)" ] && tmpcu_=$(cygpath -m $tmpcu_) && tmpo_=$(cygpath -m $tmpo_)
     test_cmd $nvcc $nvccflags "$@" $NVCC_C $(nvcc_o $tmpo_) $tmpcu_
+    test_cmd xxd
 }
 
 check_nvcc() {
diff --git a/fftools/cmdutils.c b/fftools/cmdutils.c
index fe424b6a4c..233c1361bd 100644
--- a/fftools/cmdutils.c
+++ b/fftools/cmdutils.c
@@ -1149,7 +1149,7 @@ static void print_program_info(int flags, int level)
         av_log(NULL, level, " Copyright (c) %d-%d the FFmpeg developers",
                program_birth_year, CONFIG_THIS_YEAR);
     av_log(NULL, level, "\n");
-    av_log(NULL, level, "%sbuilt with %s\n", indent, CC_IDENT);
+    // av_log(NULL, level, "%sbuilt with %s\n", indent, CC_IDENT);
 
     av_log(NULL, level, "%sconfiguration: " FFMPEG_CONFIGURATION "\n", indent);
 }
diff --git a/fftools/ffprobe.c b/fftools/ffprobe.c
index 11e3cbd6c2..a65b5aa191 100644
--- a/fftools/ffprobe.c
+++ b/fftools/ffprobe.c
@@ -3159,7 +3159,7 @@ static void ffprobe_show_program_version(WriterContext *w)
     print_str("version", FFMPEG_VERSION);
     print_fmt("copyright", "Copyright (c) %d-%d the FFmpeg developers",
               program_birth_year, CONFIG_THIS_YEAR);
-    print_str("compiler_ident", CC_IDENT);
+    // print_str("compiler_ident", CC_IDENT);
     print_str("configuration", FFMPEG_CONFIGURATION);
     writer_print_section_footer(w);
 
diff --git a/libavcodec/libx264.c b/libavcodec/libx264.c
index 4ddc4973a4..1e82e42fb4 100644
--- a/libavcodec/libx264.c
+++ b/libavcodec/libx264.c
@@ -33,7 +33,7 @@
 #include "atsc_a53.h"
 
 #if defined(_MSC_VER)
-#define X264_API_IMPORTS 1
+// #define X264_API_IMPORTS 1
 #endif
 
 #include <x264.h>
